yoojin_lee/LV_1/두-정수-사이의-합.js

```diff
@@ -0,0 +1,7 @@
+function solution(a, b) {
+    let sum = 0;
+    for (let i = Math.min(a, b); i <= Math.max(a, b); i++) {
+        sum += i;
+    }
+    return sum;
+}
```

Comment:
```suggestion
function solution(a, b) {
    let sum = 0;
    for (let i = Math.min(a, b); i <= Math.max(a, b); i++) {
        sum += i;
    }
    return sum;
}
```
오옹 저랑 풀이가 같네요~!! 잘 보고 갑니닷!